index.md

```diff
@@ -0,0 +1,29 @@
+## Malagasy singers:
+### Welcome to this list of malagasy singer. If you want to know the famous singer in madagascar, just click those link bellow.
```

Comment:
This shouldn't be a subheading. Make it a normal paragraph by removing the hashes.

Reply:
Fix the following errors:
- singer must be 'artist'.
- bellow is mispelled. Find the proper spelling.